src/server_manager/web_app/ui_components/app-root.js

```diff
@@ -594,7 +596,7 @@ export class AppRoot extends mixinBehaviors
     this.currentPage = 'intro';
     this.shouldShowSideBar = false;
 
-    this.addEventListener('RegionSelected', this.handleRegionSelected);
+    this.addEventListener('region-selected', this.handleRegionSelected);
```

Comment:
Why do you need to rename this?

Reply:
I think they convert CamelCase to kebab-case. So you would keep `RegionSelected` event, but bind with `@region-selected=`. Perhaps  try `@regionselected=` too.

Did `@RegionSelected=` not work?

Reply:
ping